### ISSUE TITLE
an issue about Correct Evidence Regularization 

### ISSUE THREAD

**Issue by liu311** [pandeydeep9/EvidentialResearch2023#2]
Is the evidence limited to the range of 0-1 in the experiment? if not, the Correct Evidence Regularization term may get a negative loss. I am confuzed about this issue.

**Comment by pandeydeep9** [comment]
The evidence needs to be non-negative (>0). It can take any value from 0 to infinity. Yes, the correct evidence regularization can take a negative value. The overall loss ranges from - infinity to infinity.

**Comment by liu311** [comment]
In my opinion, isn't it the case that loss should not be non-negative in deep learning experiments? It is uncommon for the loss to be non-negative . Is there any relevant literature where the range of loss is from - infinity to infinity ?

**Comment by pandeydeep9** [comment]
The gradient of the proposed regularization is guaranteed to be bounded in the range [0,1] (see theorem 3). So there should not be any issue regarding learning/model parameter updates. Simply, the maximum gradient update at the logits layer is bounded in the range [0,1] even if the correct evidence regularization value is in the range (-infinity, 0).

 I'm not sure if there exists other relevant literature. 

If you want to ensure the loss is non-negative, you could try using the exponential of the correct evidence regularization term. But that may slightly impact the range of the gradients.
